tox.ini

```diff
@@ -1,16 +1,17 @@
 [tox]
 envlist =
-    py{27, 36, 37}-django111
-    py37-flake8
+    py{27, 36, 37, 38}-django111
+    py38-flake8
 
 [flake8]
 max-line-length = 99
 
 [testenv]
+download = True
```

Comment:
Why is this needed?

Reply:
The py36 travis build failed. So I have added it to get the latest versions in the venv.